### PR TITLE
nixos/doc: convert Chapter 49, 50, 51, 52, 53, 54 to CommonMark

### PR DESCRIPTION
**Pull request by bobby285271**
###### Motivation for this change

- [Related GitHub project](https://github.com/NixOS/nixpkgs/projects/37)
- [Related Discourse post](https://discourse.nixos.org/t/happy-docsgiving/10214/28)

###### Things done

Do conversion of six files from DocBook to CommonMark per [RFC 72](https://github.com/NixOS/rfcs/blob/master/rfcs/0072-commonmark-docs.md).

- `nixos/doc/manual/administration/cleaning-store.xml`
- `nixos/doc/manual/administration/control-groups.xml`
- `nixos/doc/manual/administration/logging.xml`
- `nixos/doc/manual/administration/rebooting.xml`
- `nixos/doc/manual/administration/service-mgmt.xml`
- `nixos/doc/manual/administration/user-sessions.xml`

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Relase notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: nixos/doc/manual/administration/service-mgmt.chapter.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+interested in, you\'d probably need only to use
+`services.#name#.enable = true;`. These services are defined in
+Nixpkgs\' [ `nixos/modules/` directory
+](https://github.com/NixOS/nixpkgs/tree/master/nixos/modules). In case
+the service is simple enough, the above method should work, and start
+the service on boot.
+
+*User* systemd services on the other hand, should be treated
+differently. Given a package that has a systemd unit file at
+`#pkg-out#/lib/systemd/user/`, using [](#opt-systemd.packages) will
```
Comment (Mic92) [inline nixos/doc/manual/administration/service-mgmt.chapter.md:99]: Is link this correct?

File: nixos/doc/manual/administration/service-mgmt.chapter.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+
+## Interacting with a running systemd {#sect-nixos-systemd-general}
+
+The command `systemctl` is the main way to interact with `systemd`. The
+following paragraphs demonstrate ways to interact with any OS running
+systemd as init system. NixOS is of no exception. The [next section
+](#sect-nixos-systemd-nixos) explains NixOS specific things worth
+knowing.
+
+Without any arguments, `systmctl` the status of active units:
```
Comment (Mic92) [inline nixos/doc/manual/administration/service-mgmt.chapter.md:23]:
```suggestion
Without any arguments, `systemctl` the status of active units:
```

File: nixos/doc/manual/administration/rebooting.chapter.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+quickly reboot the system using `kexec`, which bypasses the BIOS by
+directly loading the new kernel into memory:
+
+```ShellSession
+# systemctl kexec
+```
+
+The machine can be suspended to RAM (if supported) using `systemctl
+  suspend`, and suspended to disk using `systemctl
+  hibernate`.
```
Comment (Mic92) [inline nixos/doc/manual/administration/rebooting.chapter.md:27]:
```suggestion
The machine can be suspended to RAM (if supported) using `systemctl  suspend`, and suspended to disk using `systemctl hibernate`.
```

**Comment by bobby285271** [comment]
https://nixos.org/manual/nixos/unstable/#sect-nixos-systemd-nixos

https://github.com/NixOS/nixpkgs/blob/0530d9b53ccaa571b6676884a0df138008554479/nixos/doc/manual/administration/service-mgmt.xml#L117-L119

Accidently found this, which one is correct?

- ... by adding the package's attribute to `systemd.packages` and then do this ...
- ... by adding the package's attribute to `environment.systemPackages` and then do this ...

File: nixos/doc/manual/administration/service-mgmt.chapter.md (hunk 1 above)
Comment (bobby285271) [reply nixos/doc/manual/administration/service-mgmt.chapter.md]: UPD: It seems that accrding to #129121 , both `[](#opt-systemd.packages)` and `[systemd.packages](options.html#opt-systemd.packages)` produce the same result but the former one is preferred. I will do another push tomorrow.

**Comment by Mic92** [comment]
> https://nixos.org/manual/nixos/unstable/#sect-nixos-systemd-nixos
> 
> https://github.com/NixOS/nixpkgs/blob/0530d9b53ccaa571b6676884a0df138008554479/nixos/doc/manual/administration/service-mgmt.xml#L117-L119
> 
> Accidently found this, which one is correct?
> 
>     * ... by adding the package's attribute to `systemd.packages` and then do this ...
> 
>     * ... by adding the package's attribute to `environment.systemPackages` and then do this ...

`systemd.packages`

**Comment by bobby285271** [comment]
I did another push to use only URI fragment in options links, PTAL.

**Comment by bobby285271** [comment]
> > https://nixos.org/manual/nixos/unstable/#sect-nixos-systemd-nixos
> > https://github.com/NixOS/nixpkgs/blob/0530d9b53ccaa571b6676884a0df138008554479/nixos/doc/manual/administration/service-mgmt.xml#L117-L119
> > 
> > Accidently found this, which one is correct?
> > ```
> > * ... by adding the package's attribute to `systemd.packages` and then do this ...
> > 
> > * ... by adding the package's attribute to `environment.systemPackages` and then do this ...
> > ```
> 
> `systemd.packages`

Done in 9f7f6d225672bf05e714498dc6eaa68bd0800b69

**Pull request merged**